docs/en/ingest-management/tab-widgets/add-fleet-server/content.asciidoc

```diff
@@ -52,7 +52,9 @@ image::images/add-fleet-server.png[In-product instructions for adding a {fleet-s
 
 * Make sure you download an x64 architecture installation package.
 * After generating the service token, save it to a secure location. You might
-need to use this token later to scale your deployment by adding {fleet-server}s. 
+need to use this token later to scale your deployment by adding {fleet-server}s.
```

Comment:
```suggestion
want to use this token later to scale your deployment by adding additional {fleet-server}s.
```

Reply:
I'm going to add this then edit it because "adding additional" is redundant. :-)